src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayListPositioner.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import type {OverlayListDimensions, OverlayListStyle} from './types';
+
+const MIN_HEIGHT = 100;
+const INDENT_FROM_ANCHOR = 5;
+const PADDING_TO_WINDOW = 10;
+
+type VerticalCrop = {
+    dimensions: OverlayListDimensions,
+    touchesTopBorder: boolean,
+    touchesBottomBorder: boolean,
+};
+
+/**
+ * The class is responsible for calculating the position of the overlay-list on the screen when opened.
+ * Generally, it positions the list above an anchor element. Moreover the list is shifted, such that
+ * a designated element in the list is right above the anchor element.
```

Comment:
It's not always the selected element. (E.g. when nothing is selected and the first option is disabled - see example in styleguidist)